### PR TITLE
Jasperserver persistence nami

### PR DESCRIPTION
**Pull request by raquel-campuzano**
Using Wordpress as a model.

### CODE REVIEW

**Comment by sameersbn** [comment]
@raquel-campuzano can you please rebase this PR to the master?

File: README.md [hunk 1]
```diff
@@ -81,29 +94,32 @@ services:
   application_data:
     image: bitnami/jasperserver:latest
     volumes:
-      - /bitnami/jasperserver
-      - /bitnami/tomcat
+      - /path/to/jasperserver-persistence:/bitnami/jasperserver
+      - /path/to/tomcat-persistence:/bitnami/tomcat
```
Comment (sameersbn) [inline README.md]: I am not a 100% if a volume mount is required for tomcat. but in uncertainty i would default to mounting a volume. Could you please add this volume in the snippet at https://github.com/raquel-campuzano/bitnami-docker-jasperserver/#run-the-application-using-docker-compose and also the `docker-compose.yaml` file in the repo.

File: README.md [hunk 2]
```diff
@@ -81,29 +94,32 @@ services:
...
     entrypoint: 'true'
     mounts:
-      - /your/local/path/bitnami/jasperserver:/bitnami/jasperserver
-      - /your/local/path/bitnami/tomcat:/bitnami/tomcat
+      - /path/to/bitnami/jasperserver:/bitnami/jasperserver
+      - /path/to/bitnami/tomcat:/bitnami/tomcat
```
Comment (sameersbn) [inline README.md]: Looks like this `mounts` block here is a error. Actually this whole snippet is wrong. Could you rewrite the snippet based on the `docker-compose.yml` file. It's weird that no one noticed it.

**Comment by raquel-campuzano** [comment]
The `docker-compose.yml` file had some wrong snippets in the "Persistence your application" section. I removed this snippets and rewritten it using the`docker-compose.yml` as reference.

File: README.md [hunk 3]
```diff
@@ -203,10 +223,13 @@ To backup your application data follow these steps:
 
 2. Copy the JasperReports data folder in the host:
 
+```bash
+$ docker cp /your/local/path/bitnami:/bitnami/jasperserver
+```
+=======
```
Comment (sameersbn) [inline README.md]: this block should go.

File: README.md [hunk 4]
```diff
@@ -112,10 +130,12 @@ Bitnami provides up-to-date versions of JasperReports, including security patche
 
 1. Get the updated images:
 
-  ```
+```bash
+$ docker pull bitnami/jasperserver:latest
+```
```
Comment (sameersbn) [inline README.md]: above block should go

**Pull request merged**